pkg/fleet/installer/repository/repository.go

```diff
@@ -481,3 +505,139 @@ func (l *link) Delete() error {
 	l.packagePath = nil
 	return nil
 }
+
+func buildFileMap(rootPath string) (map[string]struct{}, error) {
+	files := make(map[string]struct{})
```

Comment:
Am I right in assuming this map would be case sensitive? On Windows it probably _shouldn't_ be case sensitive, right?

Reply:
As discussed, for simplicity we will ignore case insensitivity as an issue, as we don't expect the case to change between the OCI and the file system, and if it does, the repair will fail anyway, which is acceptable in this unexpected scenario. We can update this to explicitly do case insensitive comparisons if need be, but for now, it's fine